util/raclgen/lib.py

```diff
@@ -118,38 +122,34 @@ def parse_racl_mapping(racl_config: Dict[str, object], mapping_path: str,
         raise SystemExit('RACL group must be a string')
     if not isinstance(register_mapping, dict):
         raise SystemExit('Register mapping must be a a dict')
+    if racl_group not in racl_config['policies']:
+        raise SystemExit(f'RACL group {racl_group} not defined in RACL config')
+
+    policy_names = [
+        policy['name'] for policy in racl_config['policies'][racl_group]
+    ]
```

Comment:
I think this would be a bit cleaner as something like this:
```python
    racl_policy = racl_config['policies'].get(racl_group)
    if racl_policy is None:
        raise SystemExit(f'RACL group {racl_group} not defined in RACL config')

    policy_names = [policy['name'] for policy in racl_policy]
```

(I don't have an opinion about the variable name that I picked!)